srcpkgs/sunshine/template

```diff
@@ -0,0 +1,83 @@
+ https://github.com/LizardByte/Virtual-Gamepad-Emulation-Client/archive/8d71f6740ffff4671cdadbca255ce528e3cd3fef.tar.gz>ViGEmClient.tar.gz
+ https://github.com/LizardByte/build-deps/archive/efd3a380113e8ae98ae68cc1d73fd7c4b54b03c6.tar.gz>build-deps.tar.gz
+ https://github.com/google/googletest/archive/f8d7d77c06936315286eb55f8de22cd23c188571.tar.gz>googletest.tar.gz
+ https://github.com/moonlight-stream/moonlight-common-c/archive/cbd0ec1b25edfb8ee8645fffa49ff95b6e04c70e.tar.gz>moonlight-common-c.tar.gz
+ https://github.com/sleepybishop/nanors/archive/e9e242e98e27037830490b2a752895ca68f75f8b.tar.gz>nanors.tar.gz
+ https://github.com/FFmpeg/nv-codec-headers/archive/22441b505d9d9afc1e3002290820909846c24bdc.tar.gz>nv-codec-headers.tar.gz
+ https://github.com/LizardByte/nvapi-open-source-sdk/archive/c0f5f7b64d2ef13b1155f078a2eec156611f2415.tar.gz>nvapi-open-source-sdk.tar.gz
+ https://github.com/LizardByte/tray/archive/4d8b798cafdd11285af9409c16b5f792968e0045.tar.gz>tray.tar.gz
+ https://gitlab.freedesktop.org/wayland/wayland-protocols/-/archive/08d1c7276d41379acfea353b5c739b72d51827e2/wayland-protocols-08d1c7276d41379acfea353b5c739b72d51827e2.tar.gz>wayland-protocols.tar.gz
+ https://gitlab.freedesktop.org/wlroots/wlr-protocols/-/archive/2b8d43325b7012cc3f9b55c08d26e50e42beac7d/wlr-protocols-2b8d43325b7012cc3f9b55c08d26e50e42beac7d.tar.gz>wlr-protocols.tar.gz
+ https://gitlab.com/eidheim/Simple-Web-Server/-/archive/27b41f5ee154cca0fce4fe2955dd886d04e3a4ed/Simple-Web-Server-27b41f5ee154cca0fce4fe2955dd886d04e3a4ed.tar.gz>Simple-Web-Server.tar.gz
+ https://github.com/cgutman/enet/archive/04e27590670a87a7cd40f5a05cda97467e4e25a3.zip>enet.tar.gz"
+checksum="f811b07950a6bfd041e80cc424f95e29d4ba84a8bcdf2a61d5cdc751152c9927
+ e7bf9d090be90e419860019e6ae125bb07b6aa4d67485e70c93e3e11d41938ee
+ 27907cba8c017de220b66e036ed04a39f5eacb9182846700047420aef2f5cd0e
+ c6d9343c8a6306e1b6025bf30893f555b1f1f1a3ffe75ddd3e2e67911adc7339
+ 7ff5db23de232a39cbb5c9f5143c355885e30ac596161a6b9fc50c4538bfbf01
+ 8390c3fc51796d228577c6887bfa7c3afda42cc18e5967187951c3e0ad343458
+ fee62ecffef3b456e789852787cf534f4712e1a7973bcdd5763baad4b7d2e879
+ dc28f764fbc72f6ac2a96fcace252e6ecc84b308b8d4400820203cc901bae596
+ d97efcff3bb86472dd4f602044903a3527a2c850eeebfb9b039ab50d2a237223
+ 298e90d80c146b07fd71040d08e5184b0baca362258bf26033d7c9b80cdac11f
+ d511c8e6ac8fa356cb327e800e7a1b37ecc50b11f74ae8adf0666313b9ad41e0
+ 5bfcca71ba8548bb5d7fdb6ba0ec2c883bf79deca74282c4aba75978837ac000
+ 6179016c556c630e2c1459b38f93db881955e191840e069dd4a98dd3dd1dc56e
+ bc2f1e6975aa5394683c1f82a53459cc45c3cd7a0f9c37b8afc8d77f8818fd58"
+skip_extraction="
+ ${pkgname}-${version}.tar.gz
+ Simple-Web-Server.tar.gz
+ TPCircularBuffer.tar.gz
+ ViGEmClient.tar.gz
+ build-deps.tar.gz
+ googletest.tar.gz
+ moonlight-common-c.tar.gz
+ nanors.tar.gz
+ nv-codec-headers.tar.gz
+ nvapi-open-source-sdk.tar.gz
+ tray.tar.gz
+ wayland-protocols.tar.gz
+ wlr-protocols.tar.gz
+ enet.tar.gz"
```

Comment:
some of these are packaged, it would be good to use the packaged version instead